pkg/autoscaler/multiscaler.go

```diff
@@ -35,6 +35,10 @@ const (
 	// seconds while an http request is taking the full timeout of 5
 	// second.
 	scaleBufferSize = 10
+
+	// SampleSize is the times to scrape metrics per second.
+	// TODO(yanweiguo): tuning the sample size
+	SampleSize = 3
```

Comment:
maybe rename it to SamplesPerSecond as to better describe the time frame

Reply:
Done.